### ISSUE TITLE
NEW BUG: Bad Cover IMAGE maxsize if i use Video fixed size

### ISSUE THREAD

**Issue by Darknior** [RetroPie/EmulationStation#379]
https://github.com/RetroPie/EmulationStation/commit/e52e63cf9cbbda6899a8bfd7b8146db9a4ac899f

I'm not sure it's your commit @tomaz82 that create this new bug for me, but now i have a strange image problem.
If i understand fine you make it to fix the problem with the THUMBNAIL that don't use the VIDEO FIXED SISE if i want to show it before video with OMX Player. No problem if i use MAXSIZE for the video.

But today i update my ES and found a new bug, all my covers IMAGE are with the same size than the video ! I don't use THUMBNAIL for the moment and if i understand fine my covers take the same size than the video to prevent if i want to show them before the video. the problem was i don't want to do it, i want to show the cover and the video, and cover at full size not 640x480.
If i add a THUMBNAIL to my gamelist.xml ES will use it to replace the video and then not put a bad size to my cover.
Now all people that want to show COVER IMAGE + VIDEO will have the two at same size :(
Or they must add THUMBNAIL to their XML.

OR i found an other solution that i don't like, use MAXSIZE again to the video to let my covers using MAXSIZE too. But if i fix the VIDEO SIZE ES will use this size for my COVER IMAGE too :(

Is it possible to fix it please ?
Thansk

**Comment by tomaz82** [comment]
Are you saying, that if you use <size> both the video and the image is stretched to that size?

**Comment by Darknior** [comment]
If i use SIZE on VIDEO the IMAGE will have the same size.
If i use MAXSIZE on VIDEO the image will use the video or his MAXSIZE.
If i have a THUMBNAIL the SIZE if forced to it but we can't see it like a problem because it's what we want to show it before the video.

The problem was you use the SIZE or RATIO on IMAGE also on the alone image. You would to do it only on the image used to show before the video.

![test](https://user-images.githubusercontent.com/3703317/36432305-29ea6932-165a-11e8-9e5d-4723fa3a960b.jpg)

**Comment by Darknior** [comment]
```
			<video name="md_video">
				<origin>0 0</origin>
				<pos>0.3833 0.242</pos>
				<size>0.3344 0.4452</size>
				<showSnapshotNoVideo>true</showSnapshotNoVideo>
				<showSnapshotDelay>false</showSnapshotDelay>
				<delay>0.1</delay> <!-- Second -->
				<zIndex>50</zIndex> <!-- 30 by Default forced to 5000 by OMX -->
			</video>
```
Not working fine force the size to the <image> too :(

with `<maxSize>0.3344 0.4452</maxSize>` working fine, image use his maxsize but i can fix all my videos like i want :(

**Comment by tomaz82** [comment]
Yes I see, that is not easely fixed due to how ES handles it's resources, it only loads the image once, and now it's used in 2 different places, at 2 different sizes, which causes a problem. No other theme has done this before. Usually you replace the md_image with md_video when you do the video view. I wont promise anything, but I'll see if there is any way to fix this,

**Comment by Darknior** [comment]
A quick fix was to make what you do only on thumbnail only use for the video replace.
Image is used for the cover, and i'm not alone to do it, but others don't see it for the moment because they don't use the ES-DEV version.
Fix the thumnail size like video is perfect. We can't use it anywhere.
But please don't link the video and image size. If the guy want to do it himself he use the maxsize option that works fine.

**Comment by tomaz82** [comment]
Show me any other theme that does this.

**Comment by Darknior** [comment]
All the theme that show a cover and a video at different place.
I'm not alone. And it is not the problem it is really a bug we don't have before.
We have thumbnail to replace video, it is not cool to resize the image too.

For exemple Comicbook show video + Cover : (in video mode like me, not detail)

![test](https://user-images.githubusercontent.com/3703317/36436458-96cdac08-1664-11e8-87b0-b8bf158f94a2.jpg)

Artbook do it too : 

![test2](https://user-images.githubusercontent.com/3703317/36436507-bb551c5a-1664-11e8-819a-3060f553286f.jpg)

And some others ... i will not try them all ... but when the users will update they're ES with the future ES stable release, all these skin will be crappy :(

**Comment by tomaz82** [comment]
I gotta check this, there has not been any changes as far as I know that would cause this, the thumbnail code has always loaded the image if no thumbnail existed. So I have a hard time to see how this would be a "new bug"

**Comment by Darknior** [comment]
Thanks a lot
For me it's NEW because this morning before update ES i don't have this bug and i change nothing on my skin.

**Comment by joolswills** [comment]
@Darknior No need for things like "NEW BUG" in the title. Also, please skip the drama - eg "all these skin will be crappy :(". I am getting a little tired of your github contributions. Also if you like a feature request, please use the built in github reaction button, rather than spamming an issue with your excitement etc.

**Comment by Darknior** [comment]
@joolswills I write "new" because the week before, the ES DEV version has not this bug. It's for help, but i can only write bug.
I'm not very strong in english, some time i use a bad word, i write crappy but i verify and i want to write break or something like that. Because yes if a skin use size and not maxsize the image will have a bad size.
When i like something i write nothing i only use the 👍 ... i write when i want to explain some thing and i take time to make videos and photos to explain better than with my bad english.
So few people report bug fine and try to help developers ... i do the best i can.
I really don't understand why you write me this ?!
